README.md

```diff
@@ -8,32 +8,22 @@ The goal of this project is to provide a simple application where the user can c
 ## Setting up the environment
 To run the application's backend part you need to have the following things set up:
 
-1. Create a database with the help of MySQL.
+1. Create a a database schema with the help of MySQL
 
-2. You should set up the following environmental variables:
+2. Ask us for the `debug.keystore` file (temporary method during development), which you need to put in your Users/"yourusername"/.android folder (in case of windows)
 
+3. Use "dev" as an @ActiveProfiles annotation in the main class (import the library, if necessary)
```

Comment:
this should not be a separate step, but should be added to the code in dev branch